signer/src/storage/model.rs

```diff
@@ -297,8 +456,6 @@ pub struct SweptWithdrawalRequest {
     /// The transaction ID of the bitcoin transaction that swept out the
     /// funds to the intended recipient.
     pub sweep_txid: BitcoinTxId,
-    /// The bitcoin transaction fulfilling the withdrawal request.
-    pub sweep_tx: BitcoinTx,
```

Comment:
I couldn't see that this needed to be used anywhere so I removed it to avoid the extra I/O. But if it's needed later for validation etc. then we can easily add it back.